docs/identities/get-started/sign-in.mdx

```diff
@@ -0,0 +1,91 @@
+---
+id: sign-in
+title: Implementing user login
+sidebar_label: Sign in
+---
+
+# Sign in
+
+```mdx-code-block
+import Tabs from "@theme/Tabs"
+import TabItem from "@theme/TabItem"
+import { getSdkUrl } from "@site/src/hooks"
+import CodeBlock from "@theme/CodeBlock"
+import { FrameworkCodeTabs } from '@site/src/components/GuidesComponents'
+import jsLogin from '!!raw-loader!./_common/code-examples/js/login.js'
+import goLogin from '!!raw-loader!./_common/code-examples/go/login_handler.go'
+import nextjsLogin from '!!raw-loader!./_common/code-examples/nextjs/login.tsx'
+```
+
+The login flow authenticates existing users in your application. This guide shows how to implement a secure login process that
+authenticates users and creates sessions.
+
+```mdx-code-block
+<FrameworkCodeTabs>
+<TabItem value="javascript">
+```
+
+```mdx-code-block
+<CodeBlock language="js"> {jsLogin} </CodeBlock>
+```
+
+```mdx-code-block
+</TabItem>
+<TabItem value="nextjs">
+```
+
+This checks if the user has an active session and redirects to the Ory login page if needed.
+
+```tsx
+export default function Page() {
+  const [session, setSession] = useState<Session | null>(null)
+
+  useEffect(() => {
+    // Check if the user is authenticated
+    const checkSession = async () => {
+      try {
+        const session = await ory.toSession()
+        setSession(session)
+      } catch (error) {
+        // No valid session found, redirect to Ory login
+        window.location.href = `${basePath}/ui/login`
+      }
+    }
+
+    checkSession()
+  }, [])
+
+  return (
+    <>
+      <pre>{JSON.stringify(session, null, 2)}</pre>
+    </>
+  )
+}
+```
+
+```mdx-code-block
+</TabItem>
+<TabItem value="go">
+```
+
+This checks for an active session and redirects to login if needed.
+
+```mdx-code-block
+<CodeBlock className="language-go" title={"login_handler.go"}>{goLogin}</CodeBlock>
+
+```
+
+```mdx-code-block
+</TabItem>
+</FrameworkCodeTabs>
+```
+
+## After successful login
+
+Ory:
+
+1. Creates a session for the user
+2. Sets a secure session cookie in the browser
+3. Redirects the user to the specified return URL or default location
+
+Your application should then check for the presence of this session cookie to determine if a user is authenticated.
```

Comment:
This section is a bit confusing to me - it is there to explain the process that ory performs after the user entered their credentials correctly?